### PR TITLE
Changed icons to font icons

### PR DESCRIPTION
**Pull request by lkduerig**

### CODE REVIEW

File: config.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@
 css_dir = "/public/css"
 sass_dir = "/public/sass"
 images_dir = "public/images"
+fonts_dir = "public/fonts"
```
Comment (amolnw2778) [inline config.rb]: Isn't the backslash needed for images and fonts directories?

File: public/components/people/peopleFilter.html [hunk 2]
```diff
@@ -1,9 +1,19 @@
- <div class="searchPeopleResult filter filter--people " id="SearchPeople" ng-hide="showAddPeopleForm" >
-      <div class="filter--controls" id="people-filter">
-          <input type="text" class="form-control" placeholder="Search user" ng-model="searchPeople" />
-          <input type="submit" value="Search" class="search-submit sprite-icons-btn-search btn--image btn-primary" ng-click="getPeopleData();">
-          <span class="add-user filter--right">
-            <button ui-sref="PeopleForm" class="pull-right btn--image sprite-icons-btn-add-user" title="Add user">Add</button>
-          </span>
-      </div>
+<div class="searchPeopleResult filter filter--people clearfix" id="SearchPeople"
+     ng-hide="showAddPeopleForm">
```
Comment (amolnw2778) [inline public/components/people/peopleFilter.html]: This can be in one line.

File: public/components/people/peopleFilter.html [hunk 3]
```diff
@@ -1,9 +1,19 @@
...
+  <div class="filter--controls" id="people-filter">
+    <input type="text" class="form-control search__text"
+           placeholder="Search user" ng-model="searchPeople"/>
```
Comment (amolnw2778) [inline public/components/people/peopleFilter.html]: This too can be in one line. Also remove closing slash, not required in HTML5.

File: public/components/people/peopleFilter.html [hunk 4]
```diff
@@ -1,9 +1,19 @@
...
+    <button type="submit" value="Search"
+            class="form-control search__submit icon icon--medium icon-search btn--image btn-primary"
+            ng-click="getPeopleData();"><span class="hide-text">Search</span>
+    </button>
+    <span class="filter--right">
+      <span class="add-user">
+        <button ui-sref="PeopleForm"
+                class="form-control btn--image icon icon--large icon-user-plus"
+                title="Add user"><span class="hide-text">Add</span></button>
```
Comment (amolnw2778) [inline public/components/people/peopleFilter.html]: So much tags. Please try to minimize them to one or max two.

File: public/components/people/peopleList.html [hunk 5]
```diff
@@ -1,35 +1,36 @@
...
-          </tr>
-        </thead>
+<div id="person-list" class="list--person">
+  <table class="table">
+    <thead>
+      <tr>
+        <td class="sortable">
+          <a href="#" ng-click="sortType = 'name'">
+            Name
+            <span ng-show="sortType == 'name'" class="icon icon-move-up fa-caret fa-caret-up"></span>
```
Comment (amolnw2778) [inline public/components/people/peopleList.html]: Strict check please!

File: public/components/people/peopleList.html [hunk 6]
```diff
@@ -1,35 +1,36 @@
...
+          </a>
+        </td>
+        <td class="sortable">
+          <a href="#" ng-click="sortType = 'role'">
+            Role
+            <span ng-show="sortType == 'role'" class="icon icon-move-down fa-caret fa-caret-down"></span>
```
Comment (amolnw2778) [inline public/components/people/peopleList.html]: Strict check please!

File: public/components/projects/projects.js [hunk 7]
```diff
@@ -117,6 +117,17 @@ angular.module('rmtProjectController', [])
...
   ])
 
+  // @todo projectsFilter and peopleFilter contain similar templates, probably should be refactored to one template.
+  .directive('projectFilter', [
+    function () {
+      return {
+        restrict: 'E',
+        replace: 'true',
+        templateUrl: '/components/projects/projectsFilter.html'
+      }
```
Comment (amolnw2778) [inline public/components/projects/projects.js]: Missing semicolon at the end. Not mandatory but good to have.

File: public/components/projects/projectsFilter.html [hunk 8]
```diff
@@ -0,0 +1,19 @@
+<div class="searchProjectResult filter filter--project clearfix" id="SearchProject"
+     ng-hide="showAddProjectForm">
```
Comment (amolnw2778) [inline public/components/projects/projectsFilter.html]: One line

File: public/components/projects/projectsFilter.html [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+  <div class="filter--controls" id="project-filter">
+    <input type="text" class="form-control search__text"
+           placeholder="Search project" ng-model="searchProject"/>
```
Comment (amolnw2778) [inline public/components/projects/projectsFilter.html]: One line. Remove closing slash.

File: public/components/projects/projectsFilter.html [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+    <button type="submit" value="Search"
+            class="form-control search__submit icon icon--medium icon-search btn--image btn-primary"
+            ng-click="getProjectData();"><span class="hide-text">Search</span>
+    </button>
+    <span class="filter--right">
+      <span class="add-user">
+        <button ui-sref="PeopleForm"
+                class="form-control btn--image icon icon--large icon-user-plus"
+                title="Add user"><span class="hide-text">Add</span></button>
```
Comment (amolnw2778) [inline public/components/projects/projectsFilter.html]: Try to minimize tags to one or two max.

File: public/components/projects/projectsList.html [hunk 11]
```diff
@@ -1,39 +1,24 @@
...
-  <div class="alert alert-{{message.status}}" role="alert" ng-if="message.msg">{{message.msg}}</div>
-      <div class="searchProjectResult filter filter--projects" id="SearchProject">
+<div id="project-list" class="list--project">
+  <table class="table">
+    <thead>
+    <tr>
+      <td class="sortable">
+        <a href="#" ng-click="sortType = 'name'">Name
+          <span ng-show="sortType == 'name'"
+                class="icon icon-move-down fa-caret fa-caret-down"></span>
```
Comment (amolnw2778) [inline public/components/projects/projectsList.html]: one line

File: public/components/projects/projectsList.html [hunk 12]
```diff
@@ -1,39 +1,24 @@
...
-          </div>
-        </div>
-      </div>
-
-    </section>
\ No newline at end of file
+    <tbody>
+    <tr ng-repeat="project in projects | orderBy:sortType:sortReverse | filter:searchProject">
+      <td class="project__name"><a
+              ui-sref="ProjectsForm({id:'{{project._id}}'})">{{project.name}}</a>
```
Comment (amolnw2778) [inline public/components/projects/projectsList.html]: One line.

File: public/components/projects/projectsList.html [hunk 13]
```diff
@@ -1,39 +1,24 @@
...
-            </table>
-
-          </div>
-        </div>
-      </div>
-
-    </section>
\ No newline at end of file
+    <tbody>
+    <tr ng-repeat="project in projects | orderBy:sortType:sortReverse | filter:searchProject">
```
Comment (amolnw2778) [inline public/components/projects/projectsList.html]: Please correct indentation.

File: public/components/projects/projectsList.html [hunk 14]
```diff
@@ -1,39 +1,24 @@
-<section class="sections section-people clearfix submission--{{message.status}}" ng-controller="ProjectController">
-  <div class="alert alert-{{message.status}}" role="alert" ng-if="message.msg">{{message.msg}}</div>
-      <div class="searchProjectResult filter filter--projects" id="SearchProject">
+<div id="project-list" class="list--project">
+  <table class="table">
+    <thead>
+    <tr>
```
Comment (amolnw2778) [inline public/components/projects/projectsList.html]: Please correct indentation.

File: public/sass/base/_variables.scss [hunk 15]
```diff
@@ -57,6 +58,7 @@ $base-font-color: $black;
 $base-font-family: $sans-font-family;
 $base-line-height: 24px;
```
Comment (amolnw2778) [inline public/sass/base/_variables.scss]: Line height should be unitless (either int or float), but not fixed pixels. See https://developer.mozilla.org/en/docs/Web/CSS/line-height

File: public/sass/components/_forms.scss [hunk 16]
```diff
@@ -32,23 +32,41 @@ fieldset {
...
-  .form-control, .btn--image {
-    line-height: sprite-icons-sprite-height(btn-search);
+  position: relative;
+
+  .form-control {
     vertical-align: middle;
   }
 
+  .btn--image {
+    color: $blue-dark
```
Comment (amolnw2778) [inline public/sass/components/_forms.scss]: Missing semicolon.

File: public/sass/components/_forms.scss [hunk 17]
```diff
@@ -32,23 +32,41 @@ fieldset {
...
+    }
   }
 
   .add-user {
+    position: relative;
+    float: right;
     border-left: 1px solid $gray-light;
-    height: sprite-icons-sprite-height(btn-add-user) + $base-unit * 2;
-    margin-top: $base-unit * -2;
-    padding-top: $base-unit * 2;
```
Comment (amolnw2778) [inline public/sass/components/_forms.scss]: Instead four different padding properties, use shorthand.

File: public/sass/components/_glyph-icons.scss [hunk 18]
```diff
@@ -0,0 +1,354 @@
...
+.icon-angular:before {
+  content: $icon-angular;
+}
+.icon-android:before {
+  content: $icon-android;
+}
+.icon-css3:before {
+  content: $icon-css3;
+}
+.icon-css_3:before {
```
Comment (amolnw2778) [inline public/sass/components/_glyph-icons.scss]: Please use one symbol for one entity. css3 and css_3 is confusing. Same for HTML5 symbol.

File: public/sass/components/_sprite-icons.scss [hunk 19]
```diff
@@ -5,12 +5,12 @@ $sprite-icons-sprite-dimensions: true;
 
 // Skill icons.
 
-.skill {
-  @include hide-text();
-  height: 32px;
-  width: 32px;
-}
+//.skill {
```
Comment (amolnw2778) [inline public/sass/components/_sprite-icons.scss]: If not needed, can be removed.

File: public/sass/components/_sprite-icons.scss [hunk 20]
```diff
@@ -5,12 +5,12 @@ $sprite-icons-sprite-dimensions: true;
...
+//  @include hide-text();
+//  height: 32px;
+//  width: 32px;
+//}
 
-@import "skills/*.png";
-$skills-sprite-dimensions: true;
-@include all-skills-sprites();
\ No newline at end of file
+//@import "skills/*.png";
```
Comment (amolnw2778) [inline public/sass/components/_sprite-icons.scss]: If not needed, can be removed.

File: config.rb (hunk 1 above)
Comment (lkduerig) [inline config.rb]: No. More likely to say the slash is not requited for css and sass directories...

File: public/components/people/peopleFilter.html (hunk 4 above)
Comment (lkduerig) [inline public/components/people/peopleFilter.html]: let it be for now. It needs to be refactored with project filter, as that's mostly duplicate code. This is a huge PR, I don't have time for testing whether those divs are important or not.

File: public/components/projects/projectsFilter.html (hunk 10 above)
Comment (lkduerig) [inline public/components/projects/projectsFilter.html]: See above comment.

File: public/sass/base/_variables.scss (hunk 15 above)
Comment (lkduerig) [inline public/sass/base/_variables.scss]: That is a compass vertical rhythm setting variable. http://compass-style.org/reference/compass/typography/vertical_rhythm/ Maybe Compass is wrong, but I'm not gonna mess with it at the moment.

File: public/sass/components/_forms.scss (hunk 17 above)
Comment (lkduerig) [inline public/sass/components/_forms.scss]: What? This is a line removal...

File: public/sass/components/_glyph-icons.scss (hunk 18 above)
Comment (lkduerig) [inline public/sass/components/_glyph-icons.scss]: There's a reason for this, and also this is generated code by icomoon.

File: config.rb (hunk 1 above)
Comment (amolnw2778) [inline config.rb]: Then it should be removed from css and sass directories.

File: public/components/people/peopleFilter.html (hunk 4 above)
Comment (amolnw2778) [inline public/components/people/peopleFilter.html]: Ok. Next time :)

File: public/components/projects/projectsList.html [hunk 21]
```diff
@@ -1,39 +1,22 @@
...
-          </div>
-        </div>
-      </div>
-
-    </section>
\ No newline at end of file
+    <tbody>
+      <tr ng-repeat="project in projects | orderBy:sortType:sortReverse | filter:searchProject">
+        <td class="project__name"><a ui-sref="ProjectsForm({id:'{{project._id}}'})">{{project.name}}</a>
+        </td>
```
Comment (amolnw2778) [inline public/components/projects/projectsList.html]: Make it one line.

File: public/components/projects/projectsList.html [hunk 22]
```diff
@@ -1,39 +1,22 @@
...
+        <td class="project__description"> {{project.description}}</td>
```
Comment (amolnw2778) [inline public/components/projects/projectsList.html]: Remove empty space before `{`. Not mandatory, good to do.

File: public/sass/base/_variables.scss (hunk 15 above)
Comment (amolnw2778) [inline public/sass/base/_variables.scss]: I know. But we can avoid that by not referring it for any selector than html or body. For now you can keep it.

File: public/sass/components/_forms.scss (hunk 17 above)
Comment (amolnw2778) [inline public/sass/components/_forms.scss]: Sorry commented at removed lines. Please refer this for below lines.

File: public/sass/base/_variables.scss (hunk 15 above)
Comment (lkduerig) [inline public/sass/base/_variables.scss]: I don't think I ever refer to it anywhere. It is something Compass uses internally to calculate ems. If we change that, we are probably going to have to get rid of vertical rhythm or something.

File: config.rb [hunk 23]
```diff
@@ -4,9 +4,10 @@
 
 # Set this to the root of your project when deployed:
 http_path = "/"
-css_dir = "/public/css"
-sass_dir = "/public/sass"
+css_dir = "public/css"
+sass_dir = "public/sass"
 images_dir = "public/images"
+fonts_dir = "public/fonts"
 javascripts_dir = "js"
```
Comment (amolnw2778) [inline config.rb]: If assets are in `public` then why not `js` directory too in `public`?

File: public/sass/components/_forms.scss (hunk 17 above)
Comment (lkduerig) [inline public/sass/components/_forms.scss]: No they can't, rhythm padding doesn't just throw 0.25em on there. It's a vertical rhythm function, that 0.25 refers to a different calculation that Compass does, and only affects top and bottom padding.

File: config.rb (hunk 23 above)
Comment (lkduerig) [inline config.rb]: There's a lot of directory cleanup to do here, like the practically empty app directory. I'm not about to start that now.

Comment (lkduerig) [inline config.rb]: I might mention it's Friday and I'm totally exhausted, I haven't even eaten today. I'm not gonna start refactoring our directory structure. Either have some other complaint or merge it and move along!

File: public/components/people/peopleList.html [hunk 24]
```diff
@@ -1,35 +1,36 @@
...
-  </div>
+    <tbody>
+      <tr ng-repeat="person in persons | orderBy:sortType:sortReverse | filter:searchPeople">
+        <td class="nowrap person__name"><a ui-sref="PeopleForm({id:'{{person._id}}'})">{{person.name}}</a></td>
+        <td>
+          <div class="person__role">{{person.role}}</div>
+          <ul class="person__skills skills-icons list--horizontal">
+            <li ng-repeat="skill in person.skills" title="{{skill}}" class="skill-icon">
+              <span class="icon icon--medium icon-{{skill | lowercase | snakecase}}"></span>
+              <span class="hide-text">{{skill}}</span>
```
Comment (amolnw2778) [inline public/components/people/peopleList.html]: If text is hidden why do we even need that?

File: public/sass/base/_variables.scss (hunk 15 above)
Comment (amolnw2778) [inline public/sass/base/_variables.scss]: How about `$base-line-height: 1.5;` and `$base-rhythm: 24px;`?

File: public/components/people/peopleList.html (hunk 24 above)
Comment (lkduerig) [inline public/components/people/peopleList.html]: For screen readers.

File: public/sass/base/_variables.scss (hunk 15 above)
Comment (lkduerig) [inline public/sass/base/_variables.scss]: What difference does that make, when we are still having to use a px variable for $base-rhythm?

**Pull request merged**